src/views/CurrencyExchange.vue

```diff
@@ -0,0 +1,55 @@
+<script>
+</script>
+
+<template>
+        <div class="text-center"><h1 class="display-7">Currency Exchange</h1></div>
+        
+        <div class="text-center">
+        <div class="btn-group">
+        <select class="form-select form-select-lg mb-3" style="width:300px">
+        <option selected>Select Starting Currency</option>
+            <option value="1">USD</option>
+            <option value="2">EUR</option>
+            <option value="3">CAD</option>
+        </select>
+        &ensp;&ensp;
```

Comment:
Just kinda curious, whe use two half width spaces instead of one full width: `&nbsp;`?

Reply:
I was testing how much I wanted it spaced out and just didn't clean it up. Probably should've done that before the push.